docs/root/manual/getting-started/SPO-on-boarding guide.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_position: 1
+---
+
+import NetworksMatrix from '../../networks-matrix.md';
+import CompiledBinaries from '../../compiled-binaries.md'
+
+# SPO On-boarding Guide
+
+## The journey with Mithril toward `mainnet`
+
+Welcome to Mithril! Here is a description of the journey you will have to follow as a SPO in order to get ready for running a **Mithril Signer** in the `mainnet`.
+We are assuming that you are already an active SPO on the Cardano `mainnet` and at least one of the testnets `preview` or `preprod`.
+
+:warning: **Important**: Some steps are optional, some are mandatory.
```

Comment:
If that is the case, it would be convenient to tell which one are optional and which ones are mandatory (color code?)